libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/binder/map/MapBinder.kt

```diff
@@ -44,9 +45,13 @@ internal class MapBinder(
     private val mapView: MapView
 ) : UIBinder {
 
+    private val compassComponent =
+        CompassComponent(mapView, context.styles.mapCompassParams, context.systemBarsInsets)
+    private val scalebarComponent =
+        ScalebarComponent(mapView, context.styles.mapScalebarParams, context.systemBarsInsets)
+
     init {
-        mapView.compass.enabled = false
-        mapView.scalebar.enabled = false
+        mapView.compass.fadeWhenFacingNorth = false // todo remove
```

Comment:
Let us remove the init block

Reply:
Oh, right, I used it for testing purposes. It's true by default, by the way, IMO, that's something we can easily expose as well. For example, 1TAP explicitly sets it to false.

Reply:
Lets do it

Reply:
I think it should be hardcoded to false, otherwise we will move our buttons down to make up space for compass but it won't show up (if we are facing north). I'm not sure there is a good way now to understand exactly when the compass view actually shows up.